tests/Yaapii.Atoms.Tests/Scalar/TernaryTest.cs

```diff
@@ -71,5 +71,52 @@ public void ConditionFunc()
                     input => input = 2
                 ).Value() == 8);
         }
+
+        //
+
+        [Fact]
+        public void ConditionTrueOneGeneric()
+        {
+            Assert.True(
+                new Ternary<int>(
+                    new True(),
+                    6,
+                    16
+                ).Value() == 6);
+        }
+
+        [Fact]
+        public void ConditionFalseOneGeneric()
+        {
+            Assert.True(
+                new Ternary<bool, int>(
+                    new False(),
+                    6,
+                    16
+                ).Value() == 16);
+        }
+
+        [Fact]
+        public void ConditionBooleanOneGeneric()
+        {
+            Assert.True(
+            new Ternary<bool, int>(
```

Comment:
same here

Reply:
`new Ternary<bool, int>(`
i still see 2 generics here and the test name still says `One`.